src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameCollector.cs

```diff
@@ -240,6 +240,8 @@ private void SessionEnded_Handler(object sender, SessionEndEventArgs args)
                 }
             }
 
+            // Terminate the proc dump process
+            this.processDumpUtility.TerminateProcessDump();
```

Comment:
Please also validate this where no dump collection is enabled

Reply:
what if there's an exception in the sessionend will this be hit ?

---
In reply to: [235662297](https://github.com/Microsoft/vstest/pull/1849#discussion_r235662297) [](ancestors = 235662297)

Reply:
Fair, added a check to see if dump collection was enabled or not before attempting to terminate proc dump

Reply:
Added a finally block to ensure this will always execute